src/olympia/reviewers/utils.py

```diff
@@ -1671,10 +1673,28 @@ def change_or_clear_pending_rejection_multiple_versions(self):
 
     def enable_addon(self):
         """Force enable the add-on."""
+        self.file = None
         self.version = None
-        self.addon.force_enable(skip_activity_log=True)
+        # Force queryset evaluation before we enable the versions, so that the
+        # list of versions is properly recorded after.
+        files_qs = File.objects.disabled_that_would_be_renabled_with_addon().filter(
+            version__addon=self.addon
+        )
+        versions = tuple(
+            self.addon.versions(manager='unfiltered_for_relations')
+            .filter(pk__in=files_qs.values_list('version'))
+            .no_transforms()
+            .only('id', 'version')
+            .order_by('-pk')
+        )
         log.info('Sending email for %s' % (self.addon))
-        self.record_decision(amo.LOG.FORCE_ENABLE)
+        # We can't use record_decision(..., action_completed=False) like we do
```

Comment:
Not for this patch, obvs, but would it be possible to reconcile this code with `ContentActionTargetAppealApprove.process_action` at some point?

Reply:
Ah yes, that's the 3rd issue I need to file